index.html

```diff
@@ -2880,9 +2893,18 @@
           further action. The <a>user agent</a> user interface should ensure
           that this never occurs.
           </li>
-          <li>
-            <a>Fire an event</a> named <var>name</var> at <var>request</var>
-            using <a>PaymentRequestUpdateEvent</a>.
+          <li>Let <var>event</var> be the result of <a>fire an event</a> named
```

Comment:
This doesn't work. Fire an event returns a boolean, not an event.

You should instead:

- Let _event_ be the result of [creating an event](https://dom.spec.whatwg.org/#concept-event-create) using Event.
- Initialize _event_'s type attribute to _name_
- [Dispatch](https://dom.spec.whatwg.org/#concept-event-dispatch) _event_ at _target_.

Reply:
> This doesn't work. Fire an event returns a boolean, not an event.

Argh, I should have remembered that. I've used that boolean before elsewhere.